### DISCUSSION TITLE
Poll: Removal of REI Subsets

### DISCUSSION THREAD

**Discussion by shedaniel** [shedaniel/RoughlyEnoughItems#964] [Polls]
**Edit: Subsets have been removed from REI since build 518.**

---

REI has had Subsets from the early days (We are talking about 2019 days)

Since then REI has introduced the manual filtering screen and search filter rules, which allows for fast and customisable filtering.

Subsets is a feature from NEI to hide a subset of items quickly, but with a giant ugly button on top of the screen.

Subsets is seen limited usage in REI due to the aforementioned alternatives, __and I am planning for the removal of subsets in the future versions of REI__.

**Comment by Azzyypaaras** [comment]
Subnuts

**Comment by Shnupbups** [comment]
I agree with Azazel tbh

**Reply by shedaniel** [reply]
ur mum

**Reply by vini2003** [reply]
no u
